recharges/tasks.py

```diff
@@ -108,6 +108,38 @@ class Hotsocket_Get_Airtime(Task):
     """
     name = "recharges.tasks.hotsocket_get_airtime"
 
+    def normalize_msisdn(self, msisdn, country_code='',):
+        if len(msisdn) <= 5:
+            return msisdn
+        msisdn = ''.join([c for c in msisdn if c.isdigit() or c == '+'])
+        if msisdn.startswith('00'):
+            return '+' + country_code + msisdn[2:]
+        if msisdn.startswith('0'):
+            return '+' + country_code + msisdn[1:]
+        if msisdn.startswith('+'):
+            return msisdn
+        if msisdn.startswith(country_code):
+            return '+' + msisdn
+        return msisdn
+
+    def look_up_mobile_operator(self, msisdn):
+        mtn = ['+2783', '+2773']
```

Comment:
Also, some are shorter prefixes than others - so you'll need to allow for that in this code, try a short match then a longer match.